src/general/Mutations.cs

```diff
@@ -440,6 +440,47 @@ private string MutateWord(string name)
         int letterChanges = 0;
         int changes = 0;
 
+        // Case of 1-letter words, e.g. Primum B
```

Comment:
Might be worth mentioning that without these special actions single letter names can cause infinite recursion.